src/new-components/button-link/button-link.test.tsx

```diff
@@ -240,6 +240,24 @@ describe('ButtonLink', () => {
         expect(buttonLink).not.toHaveClass('align-end')
     })
 
+    it('contains a non-clickable nested span', () => {
```

Comment:
Note that this test fails, as the `onClick` function gets called anyway. I was hoping that, if `pointer-events: none` was set, then the `onClick` function would not been called, but I guess I'm wrong 😅 Trying to think of another way to test that the nested span is non-clickable... 🤔

Reply:
The reason is that in jest tests, the stylesheet is not loaded. So the `pointer-events: none` style is not applied. I took the liberty to push an update to this branch that removes the test, and it also applies the new style rule to any element inside the button, and not just to the span. There’s for instance, the icons that can possibly be inside a button.